src/components/tools/listing/Listing.js

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect, useState } from "react";
+import { useQuery } from "react-query";
+import TableView from "./TableView";
+import { getTools } from "../../../serviceFacades/tools";
+import { injectIntl } from "react-intl";
+import { withI18N } from "@cyverse-de/ui-lib";
+import messages from "../Messages";
+import DEPagination from "../../utils/DEPagination";
+
+/**
+ * Returns the tool listing component.
```

Comment:
I would say that this *is* "the tool listing component" 🤔

Reply:
Sounds reasonable. I updated the comment.